core/src/main/java/nl/nn/adapterframework/filesystem/Samba2FileSystem.java

```diff
@@ -603,4 +604,12 @@ public void remove() {
 		}
 	}
 
+	public Integer getPort() {
+		return port;
+	}
+
+	public void setPort(Integer port) {
+		this.port = port;
+	}
+
```

Comment:
Please move this code to line 516, to have it grouped with the other getters and setters.
Please also add IbisDoc.